contrib/gitian-build.sh

```diff
@@ -1,406 +1,389 @@
+    if [[ $result == "" ]]; then
+        echo "No GPG keys available..."
+        echo "Please follow this documentation: https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/generating-a-new-gpg-key"
+        exit 1
+    fi
+    mkdir -p $outputDir/sigs
 fi
 
 # Get version
-if [[ -n "$1" ]]
-then
+if [ -n "$1" ]; then
     VERSION=$1
-    COMMIT=$VERSION
-    shift
-fi
-
-# Check that a signer is specified
-if [[ $SIGNER == "" ]]
-then
-    echo "$scriptName: Missing signer."
-    echo "Try $scriptName --help for more information"
-    exit 1
-fi
 
-# Check that a version is specified
-if [[ $VERSION == "" ]]
-then
-    echo "$scriptName: Missing version."
-    echo "Try $scriptName --help for more information"
+    # Use tag or commit within repository
+    if [[ $commit == false ]]; then
+        COMMIT=v$VERSION
+    else
+        COMMIT=$VERSION
+    fi
+else
+    echo "$scriptName: Missing version, see --help for more information."
     exit 1
 fi
 
-# Add a "v" if no -c
-if [[ $commit = false ]]
-then
-	COMMIT="v${VERSION}"
-fi
-echo ${COMMIT}
-
-# Setup build environment
-if [[ $setup = true ]]
-then
-    sudo apt-get install ruby apache2 git apt-cacher-ng python-vm-builder qemu-kvm qemu-utils
-    # GIT --date=format-local support
-    MIN_GIT_VERSION=2.7.0
-    LASTEST_GIT_VERSION=2.32.0
-    if ! (echo a version ${MIN_GIT_VERSION}; git --version) | sort -Vk3 | tail -1 | grep -q git; then
-      sudo apt-get install build-essential make libssl-dev libghc-zlib-dev libcurl4-gnutls-dev libexpat1-dev gettext unzip
-      wget https://github.com/git/git/archive/v${LASTEST_GIT_VERSION}.zip -O v${LASTEST_GIT_VERSION}.zip
-      unzip v${LASTEST_GIT_VERSION}.zip
-      pushd ./git-${LASTEST_GIT_VERSION}
-      make -j "${proc}" prefix=/usr/local all
-      make -j "${proc}" prefix=/usr/local install
-      popd
-    fi
+### Setup ###
 
-    # GIT CLONE
+if [[ $setup == true ]]; then
     git clone https://github.com/dogecoin/gitian.sigs.git
     git clone https://github.com/dogecoin/dogecoin-detached-sigs.git
     git clone https://github.com/devrandom/gitian-builder.git
+
     pushd ./gitian-builder
-    if [[ -n "$USE_LXC" ]]
+
+    #Download dependencies
+    mkdir -p inputs
+    pushd inputs
+
+    download_file $ossPatchUrl $ossPatchHash
+    download_file $ossTarUrl $ossTarHash
+    download_file $macosSdkUrl $macosSdkHash
+
+    popd
+
+    #Check if apt-cacher should be enabled
+    if [ -z "$MIRROR_HOST" ] && [[ $enableCache = false ]]; then
+        cacher_option="--disable-apt-cacher"
+    fi
+
+    #Prepare containers depending of virtualization solution: lxc, docker, kvm
+    if [ "$USE_LXC" -eq 1 ]
     then
-        sudo apt-get install lxc
-        bin/make-base-vm --suite trusty --arch amd64 --lxc
+        sudo apt-get install -y lxc
+        bin/make-base-vm --suite trusty --arch amd64 --lxc $(echo $cacher_option)
```

Comment:
Related to https://github.com/dogecoin/dogecoin/pull/2501 *"Harmonize CI and Gitian to use the same toolchain"*, I suppose suite should change to bionic

Reply:
correct! Please change to `bionic`

Reply:
fixed